dotcom-rendering/src/web/components/FilterKeyEventsToggle.importable.tsx

```diff
@@ -23,6 +23,7 @@ const toggleWrapperStyles = css`
 interface Props {
 	filterKeyEvents: boolean;
 	id: 'filter-toggle-mobile' | 'filter-toggle-desktop';
+	tooltip: boolean;
```

Comment:
I don't see how this is going to work: do you not need to pass this prop into the `ToggleSwitch ` component rather the the `FilterKeyEventsToggle` ?

Reply:
Is it possible that when you were working on this that maybe you were still yarn linked to a version of Source with this hardcoded? 

As it is I think we'd need to update the version of Source development kitchen being used by DCR in the package.json of the parent repo.